### ISSUE TITLE
Cursor In-game

### ISSUE THREAD

**Issue by MrRaccoonMan** [CookiePLMonster/SilentPatchBully#17]
My mouse cursor shows up in-game and it's not in sync with the cursor with bully ( Ex. Biology)

Any fix on this?

**Comment by CookiePLMonster** [comment]
Does this happen also without the patch? I don't recall anyone mentioning that.

Also, you may be using some software to force the game to go into borderless fullscreen, like Borderless Gaming.

**Comment by MrRaccoonMan** [comment]
Nope. Was fine before the patch and no additional software was used to go borderless fullscreen
